src/FondOfSpryker/Yves/GoogleTagManager/Twig/GoogleTagManagerTwigExtension.php

```diff
@@ -61,31 +58,31 @@ class GoogleTagManagerTwigExtension extends TwigExtension
     protected $isEnabled;
 
     /**
-     * @var \FondOfSpryker\Yves\GoogleTagManager\Business\Model\DataLayer\VariableBuilderInterface
+     * @var array
      */
-    protected $variableBuilder;
+    protected $variableBuilders;
 
     /**
-     * GoogleTagManagerTwigExtension constructor
+     * GoogleTagManagerTwigExtension constructor.
      *
      * @param string $containerID
      * @param bool $isEnabled
-     * @param \FondOfSpryker\Yves\GoogleTagManager\Business\Model\DataLayer\VariableBuilderInterface $variableBuilder
+     * @param array $variableBuilders
```

Comment:
Gibt es einen Grund warum die VariableBuilder kein gemeinsames Interface haben?

Reply:
Jepp, weil getVariables() unterschiedliche Parameter erwartet.

Reply:
Dann sollte man das umbauen ;-)

Reply:
Oder für jedes Ding ein eigenes Interface. 
Oder man überlegt sich wie man ein Interface bauen könnte das für alle funktioniert. 
Injeziert werden sollte das ganze nicht in einem Array mit Keys, sondern einzeln.

Reply:
Und getContainerID zu getContainerId... Wir sind CamelCase.

Reply:
Kommt erst in einer nächsten Version